alicloud/plugin.go

```diff
@@ -42,6 +42,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"alicloud_vpc_vpn_customer_gateway": tableAlicloudVpcVpnCustomerGateway(ctx),
 			"alicloud_vpc_vpn_gateway":          tableAlicloudVpcVpnGateway(ctx),
 			"alicloud_vpc_vswitch":              tableAlicloudVpcVSwitch(ctx),
+			"alicloud_elastic_container":        tableAlicloudElasticContainer(ctx),
```

Comment:
Can we change the table name to `alicloud_cs_kubernetes` as per terraform?
And also sort the list